lib/veri_traci.ml

```diff
@@ -0,0 +1,89 @@
+open Bap.Std
+module Bap_value = Value
+open Bap_traces.Std
+open Monad.State
+open Bil.Result
```

Comment:
Don't open Bil.Result at all. This module wasn't designed to be opened in the module implementation. If you need `'a u` and `'a r`, then just reintroduce only them.